notebooks/exploring_explorer.livemd

```diff
@@ -358,23 +358,32 @@ DataFrame.select(df, &String.ends_with?(&1, "fuel"), :drop)
 
 ### Filter
 
-The next verb we'll look at is `filter`. You can pass in a boolean mask series of the same size as the dataframe:
+The next verb we'll look at is `filter`. Filter can be done by using masks, which
+are boolean lists or series, or functions that allow more complex operations.
+
+You can pass in a boolean mask series of the same size as the dataframe:
```

Comment:
I think we should not talk about masks. Masks are mostly an anti-pattern. I would remove this section. :)